tests/integration/test_events.py

```diff
@@ -684,13 +688,23 @@ def forward_request(self, method, path, data, headers):
                     auth.get("key"): auth.get("parameters"),
                     "InvocationHttpParameters": {
                         "BodyParameters": [
-                            {"Key": "key", "Value": "value", "IsValueSecret": False}
+                            {"Key": "key", "Value": "value", "IsValueSecret": False},
                         ],
                         "HeaderParameters": [
-                            {"Key": "key", "Value": "value", "IsValueSecret": False}
+                            {"Key": "key", "Value": "value", "IsValueSecret": False},
+                            {
+                                "Key": "overwrited_header",
```

Comment:
nit (applies to the other instances of this as well)

```suggestion
                                "Key": "overwritten_header",
```

Though overridden might actually suit this case even better.